### ISSUE TITLE
Crash on Android after closing app with tab group and infinite scroll

### ISSUE THREAD

**Issue by jvandijk** [FokkeZB/nl.fokkezb.infiniteScroll#48]
I've created an app with a tab group in the root window.

A single tab has infinite scroll logic. When I click the back button to close the app, I experience a crash, but only when using `$.is.detach` or `$.is.cleanup`.

It seems the tab group is already doing some cleaning up, because the code is running to an error when trying to execute `$.isIndicator.hide()`

The log shows the following:
```
[DEBUG] TabGroup: Tabgroup is closed normally.
[WARN]  TiUIActivityIndicator: (main) [570,8340] Unable to create an activity indicator. Activity is null
```

Not using the cleanup methods solves the problem, but that's not what you want.

**Comment by FokkeZB** [comment]
Probably because cleanup calls detach and detach calls state and state... does stuff with objects already  destroyed because you close the tab/app.

The only thing however related to the activity indicator is hiding it:
https://github.com/FokkeZB/nl.fokkezb.infiniteScroll/blob/master/controllers/widget.js#L91

Try commenting out stuff to see if that solves it.

However, I must say I never use cleanup and only use detach if I reached the end and want to get rid of the infinite scroll.

**Comment by jvandijk** [comment]
It's the indicator... for sure. But the activity is already gone, because the tabs have no activity themselves.

**Comment by FokkeZB** [comment]
And does commenting out the state() call in detach() solve it?

**Comment by jvandijk** [comment]
Yep it does... But do you want that?

Verstuurd vanaf mijn iPhone

**Comment by FokkeZB** [comment]
No, but if you have time and could trace back which line exactly inside state() is causing the crash then we can maybe add a validation to catch the exception.

**Comment by jvandijk** [comment]
As I mentioned, it's the $.isIndicator.hide(). The whole activity is already gone so it cannot hide anything anymore. You want to cleanup but maybe leave out the footer related objects.

Verstuurd vanaf mijn iPhone

**Comment by FokkeZB** [comment]
Would you be able to do a PR? I don't have much time to look at it soon.

**Comment by Topener** [comment]
@jvandijk would linked PR fix it for you? (I know it is an old issue, but I'd like to know regardless)

**Comment by jvandijk** [comment]
I can functionally check this somewhere this week. Will keep you updated!

**Comment by yankovalera** [comment]
Hi, used latest code but still have an error in Java code, by doing

```
if (OS_IOS && parentSymbol.footerView === $.is) {
    parentSymbol.footerView = undefined;
}
```

It works on Android.

Regards
